src/07-projects/projects.apib

```diff
@@ -114,7 +118,10 @@ Get details for a single project.
                         + Members
                             + decision_maker
                             + member
-
+            + actuals (object) - Only available for account administrators and users with full admin access for projects.
```

Comment:
Q: if only certain people can see this, will this be `null` in the other case or will the property be optionally returned? That choice should be documented...